### DISCUSSION TITLE
如何更好的利用 Issue 和 Discussion

### DISCUSSION THREAD

**Discussion by ZoomQuiet** [LinuxSuRen/open-source-best-practice#128] [Ideas]
> 是也乎,(￣▽￣)

其实在俺来看, 都是公开免费文档型数据库而已;
不过 Issue 是单线索,
Discussions 是可定制多层线索;

从使用上, Issue 更加简单而且接口也稳定, 可以拿来当成自动注入的接口;

Discussion 因为可以进行频道定制, 反而可以给人用的更加流畅,比如说:

- [收集：开源相关的数字化运营工具](https://github.com/LinuxSuRen/open-source-best-practice/issues/111)
- [收集：国内外开源相关的编程活动](https://github.com/LinuxSuRen/open-source-best-practice/issues/124)
- ...

这种原先必须要用 前缀 分类标签进行的素材收集, 在 Discussion 中就是一个自然的专用频道了
使用时, 也不用象 Issue 中那样还要定制搜索, 而且找起来也方便?

**Comment by LinuxSuRen** [comment]
issue 确实适合相对单一的话题（bug、proposal），甚至，内容格式也可以更加规范。

discussion 方便讨论、提问等
